syntax = "proto3";

package PhoneBookSerialize;

message Date {
  int64 year = 1;
  int64 month = 2;
  int64 day = 3;
}

message Contact {
  string name = 1;
  Date birthday = 2;
  repeated string phone_number = 3;
}

message ContactList {
  repeated Contact contact = 1;
}
